Troubleshooting-Doc/TroubleshootingDoc.adoc

```diff
@@ -0,0 +1,61 @@
+= Schulich Epsilon Software Troubleshooting Document
+
+This document contains a high-level overview of Schulich Epsilon's onboard programs.
+
+== Epsilon-Onboard-Media-Control
+
+=== Overview:
+This program is Schulich Epsilon's onboard media controller.
+The program operates on a raspberry pi and runs a music player that will play songs on the pi inside the car.
+The music player is operated through a touch screen located in the center of the car.
+
+== Epsilon-Dashboard
+=== Overview:
+This program operates Schulich Epsilon's dashboard. 
+It is responsible for displaying all important information regarding the car's systems to the driver. 
+The dashboard program runs on and receives data from the main raspberry pi.
+There are 3 configurations that the dashboard can be run in:
+
+==== Race mode:
+The display mode that is used when the car is racing, the race mode dashboard only occupies one display to conserve power.
```

Comment:
Technically, the race mode on the dashboard running on one display isn't what's saving power, but it's that the other touch screen is off.

`To allow room for the backup camera.....`